cibuildwheel/_vendored/pep514tools/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Why are we vendoring this? We're also listing and using `bashlex` as a dependency in `setup.py` so can't we do the same for `pep514tools`?

Great find, btw! This makes the whole use of PEP 514 doable! I was really dreading maintaining an implementation ourselves.

Reply:
there's no release of `pep514tools` and `pip install https://github.com/zooba/pep514tools/archive/master.zip` does not work on python 2. The easiest was to vendor this.

Reply:
Ah, sorry, I hadn't tried it but I'd seen `pip install pep514tools` in the `pep514tools`'s `README`. But the project is not that active and there's no updates to expect, so we should be fine, actually. We could revisit this when dropping 2.7, though.

Reply:
Yes, or if @zooba actually make a release (but I don't think he will given the project doesn't seem active)

Reply:
Yeah, no, even then the `setup.py` would still fail? Alternatively, we can make make a PR (or a fork) that works for 2.7, but the question is if that's really worth it :-/